### PR TITLE
Add cli support for global and organization secrets

### PR DESCRIPTION
**Pull request by lafriks**
cli support for secrets added in #1027

### CODE REVIEW

**Comment by 6543** [comment]
please update branch ;)

**Comment by 6543** [comment]
@lafriks yould you invite me to your fork or the fork-org ?

so I can apply stuff too

**Comment by lafriks** [comment]
@6543 done

**Comment by lafriks** [comment]
> @lafriks yould you invite me to your fork or the fork-org ?
> 
> so I can apply stuff too

added

File: cli/secret/secret_set.go [hunk 1]
```diff
@@ -61,12 +61,34 @@ func secretUpdate(c *cli.Context) error {
...
-		out, ferr := ioutil.ReadFile(path)
-		if ferr != nil {
-			return ferr
+		out, err := os.ReadFile(path)
+		if err != nil {
+			return err
 		}
 		secret.Value = string(out)
 	}
+	if c.Bool("global") {
```
[the remark this answers is not in the document]
Comment (lafriks) [reply cli/secret/secret_set.go:70]: I was thinking about that but there are multiple use cases that have different args processing logic (info/list vs add/rm/set)

[the remark this answers is not in the document]
Comment (lafriks) [reply cli/secret/secret_set.go]: I did some more thinking and did refactor in the end, looks better imho

**Review by 6543 — APPROVED** [review]

**Pull request merged**